docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -441,6 +441,10 @@ Property Name                                      Description
 
 ``location``                                       Optionally specifies the file system location URI for
                                                    the table.
+
+``format_version``                                 Optionally specifies the format version of the Iceberg
```

Comment:
From a new user perspective I think it would be useful to have a very high-level description about what `1` and `2` actually mean.